### PR TITLE
Add workflow bbduk-star-htseq

### PR DESCRIPTION
**Pull request by JenkoB**
mege after PR235 (https://github.com/genialis/resolwe-bio/pull/235), PR238 (https://github.com/genialis/resolwe-bio/pull/238).

### CODE REVIEW

**Comment by JenkoB** [comment]
I have skipped indexing step in workflow, but in test I left indexing step because star-index is 150MB big.

**Comment by mstajdohar** [comment]
Rebase this on to the master branch.

File: resolwe_bio/descriptors/bbduk-star-htseq.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- slug: quantseq
+  name: QunatSeq workflow
```
Comment (jkokosar) [inline resolwe_bio/descriptors/bbduk-star-htseq.yml]: QuantSeq

File: resolwe_bio/descriptors/bbduk-star-htseq.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version: 0.0.1
+  description: QunatSeq workflow selection template
```
Comment (jkokosar) [inline resolwe_bio/descriptors/bbduk-star-htseq.yml]: QuantSeq typo

File: resolwe_bio/processes/alignment/star.yml [hunk 3]
```diff
@@ -219,58 +221,58 @@
 
       if [ -f Unmapped.out.mate1 ]
       then
-        mv Unmapped.out.mate1 ${READS_NAME}_unmapped.out.mate1.fastq
-        gzip ${READS_NAME}_unmapped.out.mate1.fastq
-        re-save-file unmapped_f ${READS_NAME}_unmapped.out.mate1.fastq.gz
+        mv "Unmapped.out.mate1 ${READS_NAME}_unmapped.out.mate1.fastq"
```
Comment (jkokosar) [inline resolwe_bio/processes/alignment/star.yml]: Wrong/missing quotes

File: resolwe_bio/processes/alignment/star.yml [hunk 4]
```diff
@@ -219,58 +221,58 @@
...
+        gzip "${READS_NAME}_unmapped.out.mate1.fastq"
+        re-save-file unmapped_f "${READS_NAME}_unmapped.out.mate1.fastq.gz"
       fi
 
       if [ -f Unmapped.out.mate2 ]
       then
-        mv Unmapped.out.mate2 ${READS_NAME}_unmapped.out.mate2.fastq
-        gzip ${READS_NAME}_unmapped.out.mate2.fastq
-        re-save-file unmapped_r ${READS_NAME}_unmapped.out.mate2.fastq
+        mv "Unmapped.out.mate2 ${READS_NAME}_unmapped.out.mate2.fastq"
```
Comment (jkokosar) [inline resolwe_bio/processes/alignment/star.yml]: "Unmapped.out.mate2" "${READS_NAME}_unmapped.out.mate2.fastq"

File: resolwe_bio/processes/alignment/star.yml [hunk 5]
```diff
@@ -219,58 +221,58 @@
...
-        cp ${READS_NAME}.tab.gz ${READS_NAME}_1.tab.gz
-        cp ${READS_NAME}.tab.gz ${READS_NAME}_2.tab.gz
-        cp ${READS_NAME}.tab.gz ${READS_NAME}_3.tab.gz
-        re-export ${READS_NAME}_1.tab.gz
-        re-export ${READS_NAME}_2.tab.gz
-        re-export ${READS_NAME}_3.tab.gz
+        mv ReadsPerGene.out.tab "${READS_NAME}.tab"
+        gzip "${READS_NAME}.tab"
+        re-save-file gene_counts "${READS_NAME}.tab.gz"
+        cp ${READS_NAME}.tab.gz "${READS_NAME}_1.tab.gz"
```
Comment (jkokosar) [inline resolwe_bio/processes/alignment/star.yml]: cp "${READS_NAME}.tab.gz" "${READS_NAME}_1.tab.gz"

File: resolwe_bio/processes/alignment/star.yml [hunk 6]
```diff
@@ -219,58 +221,58 @@
...
+        cp ${READS_NAME}.tab.gz "${READS_NAME}_2.tab.gz"
```
Comment (jkokosar) [inline resolwe_bio/processes/alignment/star.yml]: See above

File: resolwe_bio/processes/alignment/star.yml [hunk 7]
```diff
@@ -219,58 +221,58 @@
...
+        cp ${READS_NAME}.tab.gz "${READS_NAME}_3.tab.gz"
```
Comment (jkokosar) [inline resolwe_bio/processes/alignment/star.yml]: See above

**Comment by JenkoB** [comment]
I have corrected the changes.

**Pull request merged**